internal/sirius/ref_data.go

```diff
@@ -0,0 +1,49 @@
+package sirius
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strconv"
+)
+
+type RefDataTeamType struct {
+	Handle string `json:"handle"`
+	Label  string `json:"label"`
+}
+
+func (c *Client) TeamTypes(ctx context.Context, cookies []*http.Cookie) ([]RefDataTeamType, error) {
+	var v struct {
+		Data []RefDataTeamType `json:"teamType"`
+	}
+
+	types := []RefDataTeamType{}
```

Comment:
Generally you'd do `var types []RefDataTeamType` instead of this to avoid an allocation

Reply:
Thanks, I didn't like it when I wrote it but couldn't quite remember what I was meant to be doing. I've fixed it now.